pkgs/development/python-modules/sentinels/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "sentinels";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1cglkxph47pki4db4kjx5g4ikxp2milqdlcjgqwmx4p1gx6p1q3v";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "sentinels" ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/sentinels";
+    description = "Various objects to denote special meanings in python";
+    license = licenses.bsdOriginal;
```

License looks like [BSD3](https://github.com/vmalloc/sentinels/blob/develop/LICENSE).

```suggestion
    license = licenses.bsd3;
```

---

thanks. I've only looked at the "BSD" from PyPi